src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/InterventionCsvController.kt

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.controller
+
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import org.springframework.web.multipart.MultipartFile
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto.InterventionDTO
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.service.InterventionCsvService
+
+@RestController
+class InterventionCsvController(
+  private val csvService: InterventionCsvService
+) {
+
+  @PostMapping("/intervention/csv/uploadfile")
```

Comment:
we should align on the new api endpoints in general, but i'd expect this one to be something like `POST /intervention/add` or `intervention/new`.